Formula/yomo.rb

```diff
@@ -0,0 +1,50 @@
+class Yomo < Formula
+  desc "Streaming Serverless Framework for Low-latency edge computing applications"
+  homepage "https://yomo.run"
+  url "https://github.com/yomorun/yomo/archive/v0.5.0.tar.gz"
+  sha256 "d7180d867837af03d53ee0f50e04ae09c56f60574d8a29afd3244c66300f9c5a"
+  license "Apache-2.0"
+
+  depends_on "go" => [:build, :test]
```

Comment:
It seems from the test block that it's needed at runtime too

Reply:
yeah, this would generate go skeleton code, I think we should make `go` dep as runtime dependency.